demo/api/src/common/blocks/accordion-item.block.ts

```diff
@@ -32,6 +32,15 @@ export const AccordionContentBlock = createBlocksBlock(
     "AccordionContent",
 );
 
+export enum AccordionItemTitleHtmlTag {
```

Comment:
Out-of-scope: We could introduce a shared "SectionHeadingLevel" or "HeadingLevel" enum that's used by all blocks with headings.